### DISCUSSION TITLE
When will the tech report of Reference Model release?

### DISCUSSION THREAD

**Discussion by liangliangdeveloper** [Mikubill/sd-webui-controlnet#1589] [General]
I just wonder when will the tech report because it really fancy work and I wonder how it works.🤣

**Comment by ghost** [comment]
Tbh., I know it sounds crazy, but I wish there was an option to turn ControlNet into negative, like it should avoid this style / these lines. I would experiment with ti2 style, shuffle and reference. So far, inputting the best picture of a batch and re-running the batch with this as a style in Controlnet has been a nice way to refine a style / composition. I wonder what would happen if I always feed it with the worst input. There is so much to discover, SD is amazing

**Reply by 2blackbar** [reply]
logically, whats negative of photo of a tree? you thought about that or its just a wacky idea with no logic, even if it would work, it would probably do horrible incoherent mess of images if you give it nice coherent ones to use as negative ref.
What if you would give it garbled mess as negative reference, would it do pretty picture of... what ?

**Reply by ljleb** [reply]
There is a way to try that @neuralweights. To enable this feature, you need to manually edit `ui-config.json` by changing the value of these fields:

- `"txt2img/Control Weight/minimum": 0.0`
- `"img2img/Control Weight/minimum": 0.0`

to `-2.0`. Then, completely restart your webui.

After a very short testing session, it seems you can use tile to drive the result away from the reference image using a negative weight. Thanks a lot for this fun idea! I think it has a similar usage to negative loras.

**Reply by ljleb** [reply]
Here is an example:

<details>

<summary>Image without controlnet</summary>

![44239-3086968368-stellar night of sparkling micro explosions, end of the world, massive yzyoljxv, best quality, 8k, deep emotional catastrophe, m](https://github.com/Mikubill/sd-webui-controlnet/assets/32277961/cdee57d2-60bd-4e36-9df8-0b1a0150707c)

</details>


<details>

<summary>Image for tile controlnet</summary>

![43519-2970869253-beautiful 1girl](https://github.com/Mikubill/sd-webui-controlnet/assets/32277961/d9435468-c530-4286-bdf2-de52524c393c)

</details>

When using weight=`-0.2`, I get this:

<details>

<summary>Image with negative controlnet</summary>

![44242-3086968368-stellar night of sparkling micro explosions, end of the world, massive yzyoljxv, best quality, 8k, deep emotional catastrophe, m](https://github.com/Mikubill/sd-webui-controlnet/assets/32277961/483652a1-8e71-460b-897f-0f9db1faa4f9)

</details>

As you can see, the fireworks are no more hidden by a random person I didn't ask for in my prompt.

<details>

<summary>Generation params</summary>

```
stellar night of sparkling micro explosions, end of the world, massive yzyoljxv, best quality, 8k, deep emotional catastrophe, masterpiece
Negative prompt: worst quality low resolution draft, monochrome, movie title,
art by xynon-bad-11k-2
Steps: 30, Sampler: DPM++ 2M Karras, CFG scale: 7, Seed: 3086968368, Size: 512x512, Model hash: d7465e52e1, Model: ghostmix_v12, CFG Rescale φ: 0
```

</details>

**Reply by ljleb** [reply]
Just a note that some preprocessors like reference-only do not seem to have any effect with a negative weight. Definitely needs more testing.
